index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Web-site</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="box-area">
+                
+                   <h1><a class="Original Trombones"></a></h1>
+                   <div class="logo"></div>
+                    <br>
+                    <div>
+                    
+                        <header id="header">
+                            <div class="logo">
+                              <h1><img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="original trombones logo"></h1>
+                            </div>
+                        
+                            <nav id="nav-bar">
+                              <ul>
+                                <li><a class="nav-link" href="#features">Features</a></li>
+                                <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
+                                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+                              </ul>
+                            </nav>
+                          </header>
+                       
+                          
+                </div>
+                <br>
+                <br>
+                <div class="handle-Trombones">

```

Comment:
Please give a proper name to your classes. Classes and ids should be in lowercase because that is a basic HTML syntax. Thanks! 😄